qdrant-landing/content/articles/qdrant-1.2.x.md

```diff
@@ -0,0 +1,202 @@
+and visual data using different models. Or you might be experimenting with different models but 
+don't want to make your payloads redundant by keeping them in separate collections.
+
+![Optional vectors](/articles_data/qdrant-1.2.x/optional-vectors.png)
+
+However, up to the previous version, we requested you provide all the vectors for each point. There 
+have been many requests to allow nullable vectors, as sometimes you cannot generate an embedding or 
+simply don't want to for reasons we don't need to know.
+
+### Grouping requests
+
+Embeddings are great for capturing the semantics of the documents, but we rarely encode larger pieces 
+of data into a single vector. Having a summary of a book may sound attractive, but in reality, we 
+divide it into paragraphs or some different parts to have higher granularity. That pays off when we 
+perform the semantic search, as we can return the relevant pieces only. That's also how modern tools 
+like Langchain process the data. The typical way is to encode some smaller parts of the document and 
+keep the document id as a payload attribute.
+
+![Query without grouping request](/articles_data/qdrant-1.2.x/without-grouping-request.png)
+
+There are, however, some cases, in which we want to find the relevant parts, but no more than just 
+a single result per document. Up till now, we had to implement such a mechanism on the client side 
+and send several calls to the Qdrant engine. But that's no longer the case. Qdrant 1.2 provides 
+a mechanism for grouping requests, which can handle that server-side, within a single call to the 
+database. This mechanism is similar to the SQL `GROUP BY` clause.
+
+![Query with grouping request](/articles_data/qdrant-1.2.x/with-grouping-request.png)
+
+### Nested filters
+
+Unlike some other vector databases out there, Qdrant accepts any arbitrary JSON payload, including
+arrays, objects, and arrays of objects. You can also [filter the search results using nested 
+keys](/documentation/filtering/#nested-key), even though arrays (using the `[]` syntax). 
+
+Before Qdrant 1.2 it was actually impossible to express some more complex conditions for the
+nested structures. For example, let's assume we have the following payload:
+
+```json
+{
+  "country": "Japan",
+  "cities": [
+    {
+      "name": "Tokyo",
+      "population": 9.3,
+      "area": 2194
+    },
+    {
+      "name": "Osaka",
+      "population": 2.7,
+      "area": 223
+    },
+    {
+      "name": "Kyoto",
+      "population": 1.5,
+      "area": 827.8
+    }
+  ]
+}
+```
+
+We want to filter out the results to include the countries with a city with over 2 million citizens 
+and an area bigger than 500 square kilometers but no more than 1000. There is no such a city in 
+Japan, looking at our data, but if we wrote the following filter, it would be returned:
+
+```json
+{
+  "filter": {
+    "must": [
+      {
+        "key": "country.cities.population",
```

Comment:
Have you tried this example?

I think you would need the `[]` notation here.

`"country.cities[].population"`